src/mentionediting.js

```diff
@@ -131,14 +142,22 @@ function preventPartialMentionDowncast( dispatcher ) {
 // @param {Object} mention
 // @param {module:engine/conversion/downcastdispatcher~DowncastConversionApi} conversionApi
 // @returns {module:engine/view/attributeelement~AttributeElement}
-function createViewMentionElement( mention, { writer } ) {
+const createViewMentionElement = (feed) => ( mention, { writer } ) => {
 	if ( !mention ) {
 		return;
 	}
 
+	const feedItem = feed.find(({ id }) => id === mention._text);
+	const isPopulatedIsUndefined = mention.isPopulated === "undefined" || mention.isPopulated === undefined;
```

Comment:
They can be `true` or `undefined` strings? :sob: 

Also, do we care about `void`?

Reply:
I dont think so, this seemed to cover all the cases 🤷